chocolatey/tools/LICENSE.txt

```diff
@@ -0,0 +1,31 @@
+﻿
+Note: Include this file if including binaries you have the right to distribute.
+Otherwise delete. this file.
+
+===DELETE ABOVE THIS LINE AND THIS LINE===
```

Comment:
Should this and the content above actually be in this PR?

Reply:
No, in fact I can remove this entire file. There's a way to include the executable in the package, that's why this file is here. But I chose to go with an easier to maintain method of an automation script that downloads it instead. Thanks for noticing.

Reply:
Thank you for being so wonderfully polite :)